### PR TITLE
[Fleet] Add known issue to Fleet 7.16

### PR DESCRIPTION
**Pull request by criamico**
Adding section "Known issues" to Fleet 7.16

### CODE REVIEW

File: docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc [hunk 1]
```diff
@@ -56,6 +56,30 @@ Fleet::
...
+
+New Agent Policies get created without System integration, regardless of having the check "Collect system logs and metrics" enabled.
+
+*Workaround*
+
+You can either:
+
+* Add manually the system integration after the Agent Policy is created
+* Rename the other system integration policies
+
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc:82]:
```suggestion

===
```

**Review by dedemorton — COMMENTED** [review]
Added a suggestion to get this building.  I'll do a proper review when the preview is available.

File: docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc [hunk 2]
```diff
@@ -56,6 +56,31 @@ Fleet::
 * Improves default settings for {fleet} component templates {kib-pull}114101[#114101]
 * Report `installing` status while package install is in progress {kib-pull}111875[#111875]
 
+[discrete]
+[[known-issues-7.16.x]]
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc:60]:
```suggestion
[[known-issues-7.16.0]]
```

File: docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc [hunk 3]
```diff
@@ -56,6 +56,31 @@ Fleet::
...
+=== Known issues
+
+{kib-pull}121059[#121059]
+Error when creating an Agent Policy, seen after editing an Integration policy that is used in other Agent Policies
+[%collapsible]
+====
+
+*Details*
+
+After editing a System integration policy used by other Agent Policies, if a new Agent Policy is created, error "There is already a package with the same name" is thrown.
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc:70]: The details here are hard to follow. I can't review/edit this without understanding this better. Can you ping me and go over the details? It sounds like it's related to integration policy names needing to be unique. Is the code not smart enough to give the system integration a unique name when you try to create an agent policy?  I'm not sure what editing an existing system integration policy has to do with this problem. A 5-minute chat would probably clear this up.

**Review by dedemorton — COMMENTED** [review]
I need more details before I can review this completely. Please see my comments.

File: docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc [hunk 4]
```diff
@@ -56,6 +56,31 @@ Fleet::
...
+
+*Impact*
+
+New Agent Policies get created without System integration, regardless of having the check "Collect system logs and metrics" enabled.
+
+*Workaround*
+
+You can either:
+
+* Add manually the system integration after the Agent Policy is created
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc:80]:
```suggestion
* Add the system integration manually after creating the agent policy 
```

File: docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc [hunk 5]
```diff
@@ -56,6 +56,31 @@ Fleet::
...
+* Rename the other system integration policies
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc:81]:
```suggestion
* Rename the other system integration policies using unique names
```

File: docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc [hunk 6]
```diff
@@ -56,6 +56,31 @@ Fleet::
...
+[%collapsible]
+====
+
+*Details*
+
+After editing a System integration policy used by other Agent Policies, if a new Agent Policy is created, error "There is already a package with the same name" is thrown.
+
+*Impact*
+
+New Agent Policies get created without System integration, regardless of having the check "Collect system logs and metrics" enabled.
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc:74]:
```suggestion
New agent policies are created without the system integration, even when "Collect system logs and metrics" is selected.
```

File: docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc [hunk 7]
```diff
@@ -56,6 +56,31 @@ Fleet::
 * Improves default settings for {fleet} component templates {kib-pull}114101[#114101]
 * Report `installing` status while package install is in progress {kib-pull}111875[#111875]
 
+[discrete]
+[[known-issues-7.16.x]]
+=== Known issues
+
+{kib-pull}121059[#121059]
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc:63]: You should either use the structure from the template (if we expect to add more known issues), or remove the collapsible section. If you keep the collapsible section, it should start with this:

```
[[known-issue-issue 121059]]
.Add short description here
[%collapsible]
====
```

File: docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc (hunk 3 above)
Comment (criamico) [reply docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc:70]: Hi DeDe, sure! I'll ping you and explain what happens.

File: docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc [hunk 8]
```diff
@@ -56,6 +56,35 @@ Fleet::
...
+[%collapsible]
+====
+
+{kib-pull}121059[#121059]
+Error when creating an Agent Policy, seen after editing an Integration policy that is used in other Agent Policies
+
+
+*Details*
+
+After editing a System integration policy used by other Agent Policies, if a new Agent Policy is created, error "There is already a package with the same name" is thrown.
```
[the remark this answers is not in the document]
Comment (dedemorton) [reply docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc:74]: trying to give some details here without making this too complex.

**Review by dedemorton — COMMENTED** [review]
Added some suggestions. If you like them, click either *Commit suggestion* to add the change to your PR, or click *Add suggestion to batch* then when you're done, submit them all at once.

File: docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc [hunk 9]
```diff
@@ -56,6 +56,35 @@ Fleet::
 * Improves default settings for {fleet} component templates {kib-pull}114101[#114101]
 * Report `installing` status while package install is in progress {kib-pull}111875[#111875]
 
+[discrete]
+[[known-issues-7.16.0]]
+=== Known issues
+
+[[known-issue-issue-121059]]
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc:63]:
```suggestion
[[known-issue-121059]]
```

**Review by dedemorton — APPROVED** [review]
LGTM. Thanks again for doing this!

**Comment by dedemorton** [comment]
I think automated backports might be flaky right now. Let's make sure this gets backported to 7.16, 7.17, and 8.0. Let me know if you want me to do that. Thanks!

**Comment by criamico** [comment]
Thank you very much for your help DeDe!
